_posts/cqrs-and-es.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "I gotta tell you: CQRS doesn't imply event sourcing"
```

Comment:
the "I gotta tell you" doesn't really add anything to the title though I admit, I does add some personality. Will leave it to you to make the call.